theano/d3viz/tests/test_d3viz.py

```diff
@@ -0,0 +1,49 @@
+import numpy as np
+import os.path as pt
+import tempfile
+import unittest
+import filecmp
+import sys
+
+import theano as th
+import theano.d3viz as d3v
+from theano.d3viz.tests import models
+
+
+class TestD3Viz(unittest.TestCase):
+
+    def setUp(self):
+        self.rng = np.random.RandomState(0)
+        self.data_dir = pt.join('data', 'test_d3viz')
+
+    def check(self, f, reference=None, verbose=False):
+        tmp_dir = tempfile.mkdtemp()
+        html_file = pt.join(tmp_dir, 'index.html')
+        if verbose:
+            print(html_file)
+        d3v.d3viz(f, html_file)
+        assert pt.getsize(html_file) > 0
+        if reference:
+            assert filecmp.cmp(html_file, reference)
```

Comment:
I agree, which is why I did not compare against a reference at the beginning, but only checked if `d3viz()` terminates with an output file. In `test_formatting`, I test the `pydot` graph which is used by `d3viz90` and later visualized by javascript. I will remove the comparison again!